src/witan/send/adroddiad/pptx/slides.clj

```diff
@@ -0,0 +1,162 @@
+(ns witan.send.adroddiad.pptx.slides
```

Comment:
I think it is worthwhile making some of the magic numbers here into named defs so it is clear why you are doing 1920, 10, 30, etc